flink-cdc-cli/src/main/java/com/ververica/cdc/cli/parser/YamlPipelineDefinitionParser.java

```diff
@@ -43,6 +43,7 @@ public class YamlPipelineDefinitionParser implements PipelineDefinitionParser {
     private static final String SINK_KEY = "sink";
     private static final String ROUTE_KEY = "route";
     private static final String PIPELINE_KEY = "pipeline";
+    private static final String FLINK_CONFIG_KEY = "flink-config";
```

Comment:
I'm not tend to introduce another top level concept(section) to users, less is more, could we make user defined flink config as part fo pipeline section?

Reply:
Sorry for the late reply. Yes, I agreed with you, we should not give users too much concept. But how do we distinguish which configs belong to flink config? Use "flink.config" as the prefix to distinguish?